demos/TicTacToe/Classes/AppDelegate.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "cocos2d.h"
+
+class AppDelegate : private cocos2d::Application {
+ public:
+  AppDelegate();
+  virtual ~AppDelegate();
```

Comment:
Instead of virtual, use override.

https://en.cppreference.com/w/cpp/language/override